backend/runner/runner.go

```diff
@@ -368,7 +368,7 @@ func (s *Service) streamLogsLoop(ctx context.Context, send func(request *ftlv1.S
 		err := send(&ftlv1.StreamDeploymentLogsRequest{
 			RequestKey:    requestKey,
 			DeploymentKey: deployment.key.String(),
-			TimeStamp:     entry.Time.Unix(),
+			TimeStamp:     entry.Time.UnixMilli(),
```

Comment:
I noticed we were dropping the milliseconds in the logs from the `runner` this seems to fix that issue.

Reply:
We should be using proto [Time](https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp), not raw integers. Same with durations. There are helper functions for converting to and from Go types in `timestamppb`.